### DISCUSSION TITLE
Does windi supports nesting?

### DISCUSSION THREAD

**Discussion by mammadataei** [windicss/windicss#260] [Q&A] [answered]
Hi, we have a codebase that I want to migrate to windicss due to tailwind's jit issues. One of the most used postcss plugins in our codebase is postcss-nested. Can we migrate without any breaking? Does windi supports nesting? What about other postcss plugins?

**Comment by Kocal** [comment, marked as answer]
Hi!
Yes you can. We used WindiCSS and nested selectors before migrating to Tailwind JIT and postcss-nested. 👀

**Reply by ahku** [reply]
Nesting works fine, I can confirm. It had some problems when I was migrating but now it seems to be perfect.
